source/conf.py

```diff
@@ -0,0 +1,56 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+sys.path.insert(0, os.path.abspath('..'))
+print(os.path.abspath('..'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'Cloudant Python SDK'
+copyright = '2021, (C) Copyright IBM Corp'
```

Comment:
Is this "year, statement" a specific format required by the tool?
If not we should replace with:
```suggestion
copyright = '© Copyright IBM Corp. 2021.'
```

Reply:
The sphinx itself embraces the owner with copyright sign and closing dot.
But I could change the format! Here is the change: 9e559f7